docs/how-to-guides/closed-captioning.md

```diff
@@ -1,8 +1,52 @@
 # Closed Captioning
 
-VoiceBase can generate subtitles or closed captions for your video project, by allowing you to retrieve the transcript of your audio or video file using the SubRip Text (SRT) format.
+VoiceBase can generate subtitles or closed captions for your video project, by allowing you to retrieve the transcript of your audio or video file using the WebVTT or SubRip Text (SRT) format.
+
+No special configuration is required. All transcripts are always available in four formats: JSON word-by-word, plain text, WebVTT and SRT.
+
+## WebVTT format
+WebVTT is a W3C standard which may be used for displaying timed text within the HTML5 <track> element.
+A WebVTT file will begin with WEBVTT after an optional UTF-8 byte order mark. The timecode format used is hours:minutes:seconds.milliseconds with hours being optional and time units fixed to two zero-padded digits and fractions fixed to three zero-padded digits (00:00:00.000).  The fractional separator used is the full-stop.
+
+
+Example:
+
+```
+WEBVTT
+
+00:00:02.76 --> 00:00:05.08
+Agent: Well this is Michael
+thank you for calling A.B.C.
+
+00:00:05,08 --> 00:00:07,03
+Cable services. How may I help you today.
+
+00:00:08,28 --> 00:00:11,93
```

Comment:
The fractions of a second are incorrectly denoted with a `,` instead of a `.`. The fractions of a second should also be three digits in length.